src/graphify.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+var Sequelize = require('sequelize');
+var rc_util = require('./lib/utility.js');
+var modelsFactory = require('./lib/models.js');
+var _ = require('lodash');
+
+function getCrawlById(dbUrl, id, logsql) {
```

Comment:
`getCrawlById` is currently duplicated in `src/info.js`
What do you think about defining it in a single place?